src/main/java/com/playtika/homework2/task3/ManipulationsWithListOfStudentsInStream.java

```diff
@@ -0,0 +1,79 @@
+package com.playtika.homework2.task3;
+
+
+import com.playtika.homework2.task3.StudentStream;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class ManipulationsWithListOfStudentsInStream {
+
+    public static void main(String[] args) {
+        int[] id = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
+        String[] firstName = {"Андрей", "Павел", "Евгений", "Анастасия", "Юрий", "Ольга", "Александр", "Артем", "Денис", "Алиса"};
+        String[] lastName = {"Попков", "Жуковский", "Невмержицкий", "Бартош", "Хизюк", "Соболева", "Кондратович", "Петров", "Петров", "Иванова"};
+        int[] age = {21, 19, 22, 18, 17, 22, 19, 20, 20, 23};
+        List<StudentStream> students = new ArrayList<>();
+        int arraySize = students.size();
+
+        int countLastNames = lastName.length;
+        System.out.println("Список студентов:\n");
+        for (int i = 0; i < countLastNames; i++) {
+            StudentStream createStudent = new StudentStream();
+            createStudent.id = id[i];
+            createStudent.firstName = firstName[i];
+            createStudent.lastName = lastName[i];
+            createStudent.age = age[i];
+            students.add(createStudent);
+            System.out.println(students.get(i).firstName + " " + students.get(i).lastName + " " + students.get(i).age);
+        }
```

Comment:
use list instead of arrays:
List<Student> students = new ArrayList<>();
students.add(new Student(...));
...